nodewatcher/nodewatcher.py

```diff
@@ -163,7 +168,8 @@ def _maintain_size(asg_name, asg_client):
         return True
 
 
-def _is_stack_ready(stack_name, region, proxy_config):
+@retry(wait_fixed=30000, retry_on_result=lambda result: result is False)
```

Comment:
I'd reduce the interval, 30 seconds is too much given that the default scaling_down_period is 2 minutes. I think 15 seconds could be a good compromise.

Reply:
done!